sa/SASCM.INI

```diff
@@ -2071,25 +2071,40 @@ DATE=2020-11-11
 0AED=3,%3d% = float %1d% to_string_format %2d%
 0AEE=3,%3d% = pow %1d% base %2d% // all floats
 0AEF=3,%3d% = log %1d% base %2d% // all floats
+
+; CLEO 4 IniFiles.cleo
 0AF0=4,%4d% = read_int_from_ini_file %1s% section %2s% key %3s%
 0AF1=4,write_int %1d% to_ini_file %2s% section %3s% key %4s%
 0AF2=4,%4d% = read_float_from_ini_file %1s% section %2s% key %3s%
 0AF3=4,write_float %1d% to_ini_file %2s% section %3s% key %4s%
 0AF4=4,%4d% = read_string_from_ini_file %1s% section %2s% key %3s%
 0AF5=4,write_string %1s% to_ini_file %2s% section %3s% key %4s%
+
+; CLEO 4 FileSystemOperations.cleo
 0B00=1,delete_file %1d% ;; IF and SET
 0B01=2,delete_directory %1d% with_all_files_and_subdirectories %2d% ;; IF and SET
 0B02=2,move_file %1d% to %2d% ;; IF and SET
 0B03=2,move_directory %1d% to %2d% ;; IF and SET
 0B04=2,copy_file %1d% to %2d% ;; IF and SET
 0B05=2,copy_directory %1d% to %2d% ;; IF and SET
+
+; CLEO 4 IntOperations.cleo
 0B10=3,%3d% = %1d% AND %2d%
 0B11=3,%3d% = %1d% OR %2d%
 0B12=3,%3d% = %1d% XOR %2d%
 0B13=2,%2d% = NOT %1d%
 0B14=3,%3d% = %1d% MOD %2d%
 0B15=3,%3d% = %1d% SHR %2d%
 0B16=3,%3d% = %1d% SHL %2d%
+0B17=2,%1d% &= %2d%
+0B18=2,%1d% |= %2d%
+0B19=2,%1d% ^= %2d%
+0B1A=1,~%1d%
```

Comment:
this syntax is currently not supported by sanny. `0B1A: ~0@` will produce an error. Please add a whitespace between `~` and the parameter:
```
0B1A=1,~ %1d%
```
I'm going to address this with https://github.com/sannybuilder/dev/issues/97 but not in 3.6.1